doc/advanced_configuration.rst

```diff
@@ -75,24 +75,39 @@ dictionary within your Sphinx ``conf.py`` file :
 Establishing local references to examples
 =========================================
 
-Linking commands in your examples to their documentation is not enough.
-Sphinx-Gallery also enables you, when documenting your modules, to reference
-into examples that use that particular module.
+Sphinx-Gallery also enables you, when documenting your modules, to
+reference into the examples that use that particular class or
```

Comment:
Not 100% sure, but maybe "reference the examples" rather than "reference into the examples".